configs/advanced_server_sample_config.txt

```diff
@@ -0,0 +1,56 @@
+axon.backward_timeout: 20
+axon.forward_timeout: 10
+axon.ip: '[::]'
+axon.max_workers: 10
+axon.maximum_concurrent_rpcs: 400
+axon.port: 8091
+axon.priority.max_workers: 10
+axon.priority.maxsize: -1
+
+dataset.batch_size: 10
+dataset.block_size: 20
+dataset.data_dir: ~/.bittensor/data/
+dataset.dataset_name: default
+dataset.max_corpus_size: 10000.0
+dataset.max_datasets: 3
+dataset.no_tokenizer: false
+dataset.num_workers: 0
+dataset.save_dataset: false
+
+logging.debug: false
+logging.logging_dir: ~/.bittensor/miners
+logging.record_log: false
+logging.trace: false
+
+neuron.blacklist.stake.backward: 100
+neuron.blacklist.stake.forward: 10
+neuron.blacklist.time: 2
+neuron.blocks_per_epoch: 2
+neuron.checking: true
+neuron.clip_gradients: 1.0
+neuron.device: cuda
+neuron.full_path: ~/.bittensor/miners/default/default/advanced_server
```

Comment:
I don't think we added the paths feature back yet; I will add it to the tasks list